### DISCUSSION TITLE
Flash problems

### DISCUSSION THREAD

**Discussion by kaweng01** [tanjeffreyz/auto-maple#193] [Q&A]
When i follow the setup i cant do on the number 6 which are 6.Inside Auto Maple's main directory, open a command prompt and run

i cant even open the "main" file. when i double click it it just flash then nothing happen. I wan to ask is there anyone know how to slove this? or is there any kindheart can teach me by discord?

**Comment by SourBean** [comment]
I have also encountered the same problem, and I hope that a senior can guide us. This is a surprising project, and we all hope to be able to use it.
